internal/host/host_test.go

```diff
@@ -4723,3 +4728,12 @@ var _ = Describe("Rebooting day2", func() {
 	})
 
 })
+
+func FindEventByName(events []*common.Event, eventName string) *common.Event {
```

Comment:
can we put it in some utils package? I see it re-occurs twice

Reply:
done